pipeline/terraform/stack/elastic_pipeline_storage.tf

```diff
@@ -6,7 +6,7 @@ resource "ec_deployment" "pipeline_storage" {
   name = "catalogue-pipeline-storage-${var.pipeline_date}"
 
   region                 = "eu-west-1"
-  version                = "7.10.2"
+  version                = "7.12.0"
```

Comment:
I have done a check, and we don't seem to have any breaking problems... but probably worth double checking if you haven't.

Reply:
We only use pretty basic Elasticsearch features in pipeline storage (index/get), so I'd be quite surprised if a minor update broke anything.

Reply:
Ah, great - I suppose any slight fluctuation in querying etc would be OK too.

Reply:
I don't think we even make any queries tbh.